README.md

```diff
@@ -1,2 +1,36 @@
-# Hello-World-Project
-This is the first Microverse Project meant to test application of gitflow and es linter configuration
+#Hello-World-Project
+    this project was build using Html and css. Git, Github, and Vs Code Studio were key technological tools used in this project.
+    
+#Getting Started
+To get a local copy up and running follow these simple example steps.
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+
```

Comment:
- [ ] Kindly use the [template provided by Microverse](https://github.com/microverseinc/readme-template) to document your code. Remember that having professional code documentation will make your projects more attractive to people that want to collaborate on them and also will provide better guidance on how to interact with them. Please consider using the template and customizing it with the project information! 🤓 🛠️